### ISSUE TITLE
Allow multiple geo selection in settings

### ISSUE THREAD

**Issue by rockeynebhwani** [HTTPArchive/cwv-tech-report#5]
Currently in 'Geo' settings, you can only select one country at a time. It will be good if multi-selection is allowed so that we can easily see impact on performance in a specific geography (e.g. Shopify's performance in different geographies)

**Comment by rviscomi** [comment]
Yeah I agree this would be nice to have. I'll have to think more about ways around the charting limitations (eg maybe having one-to-many technologies to countries in a chart).
